website/payments/locale/nl/LC_MESSAGES/django.po

```diff
@@ -159,6 +167,34 @@ msgstr "%s succesvol verwerkt."
 msgid "Could not process %s."
 msgstr "%s kon niet worden verwerkt."
 
+#: admin_views.py tests/test_admin_views.py
+msgid "{} already processed."
+msgstr "{} al verwerkt."
+
+#: admin_views.py tests/test_admin_views.py
+msgid "Successfully processed {}."
+msgstr "{} is succesvol verwerkt."
+
+#: admin_views.py
+msgid "Account holder name"
+msgstr "Rekeninghouder"
+
+#: admin_views.py
+msgid "Mandate id"
+msgstr "Mandatennummer"
```

Comment:
Kenmerk machtiging

Reply:
These terms are already part of the codebase, they're just in larger pieces of text. They have been taken from official SEPA forms and I think it would be good to stay consistent.